ansible/group_vars/all.yml

```diff
@@ -3,9 +3,11 @@ www_dir: "{{ cchq_home }}/www"
 www_home: "{{ www_dir }}/{{ deploy_env }}"
 code_home: "{{ www_home }}/current"
 log_home: "{{ www_home }}/log"
-service_home: "{{ code_home }}/services"
+legacy_service_home: "{{ code_home }}/services"
+service_home: "{{ www_home }}/services"
 virtualenv_home: "{{ code_home }}/python_env"
 encrypted_root: "/opt/data"
+project: "commcare-hq"
```

Comment:
Curious to why we have a variable to store "commcare-hq"? Is it even needed? It's hard coded at quite some places in the codebase. I don't think we will call this anything apart from `commcare-hq` ever. This adds a bit of confusion by doing this by two different ways.